orderer/consensus/etcdraft/consenter_test.go

```diff
@@ -117,6 +120,52 @@ var _ = Describe("Consenter", func() {
 		})
 	})
 
+	When("the consenter is asked about join-block membership", func() {
+		BeforeEach(func() {
+			tlsCA, _ := tlsgen.NewCA()
+			confAppRaft := genesisconfig.Load(genesisconfig.SampleDevModeEtcdRaftProfile, configtest.GetDevConfigDir())
+			confAppRaft.Consortiums = nil
+			confAppRaft.Consortium = ""
+			serverCertificates = generateCertificates(confAppRaft, tlsCA, dataDir)
+			bootstrapper, err := encoder.NewBootstrapper(confAppRaft)
+			Expect(err).NotTo(HaveOccurred())
+			genesisBlockApp = bootstrapper.GenesisBlockForChannel("my-raft-channel")
+			Expect(genesisBlockApp).NotTo(BeNil())
+		})
+
+		It("identifies a bad block", func() {
```

Comment:
All of these are not really written as a ginkgo-style test.  You might instead want to look at the [table extension](https://godoc.org/github.com/onsi/ginkgo/extensions/table).

Reply:
Done